src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.HashMap;
+import java.util.Map;
+
 public class WorkWithFile {
+    private static final String NAME_OF_FIRST_LINE = "supply";
+    private static final String NAME_OF_SECOND_LINE = "buy";
+    private static final String NAME_OF_THIRD_LINE = "result";
+    private static final String COMMA = ",";
+    private static final int NAME_INDEX = 0;
+    private static final int AMMOUNT_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String entity = systemizeEntity(parseLinesFromFile(fromFileName));
+        writeToFile(entity, toFileName);
+    }
+
+    private void writeToFile(String entity, String toFileName) {
+        File file = new File(toFileName);
+        try {
+            Files.write(file.toPath(), entity.getBytes(StandardCharsets.UTF_8),
+                    StandardOpenOption.CREATE);
+        } catch (IOException e) {
+            throw new RuntimeException("File cannot be written", e);
+        }
+    }
+
+    private String systemizeEntity(String[] parsedLines) {
+        Map<String, String> consolidatedMap = new HashMap<>();
```

Comment:
let's try simplify the solution without using Map. Working with Map is a future topic in the course